cognite_toolkit/_api/data_classes.py

```diff
@@ -7,7 +7,9 @@
 __all__ = ["Variable", "Variables", "ModuleMeta", "ModuleMetaList"]
 
 from pathlib import Path
-from typing import cast
+from typing import Any, cast
+
+import pandas as pd
```

Comment:
Should we at the same time ensure that the notebook can run in pyoide? It would be great to be able to have a notebook in fusion that automatically gets the credentials (through CDFConfigTool) and then be able to install modules.

Reply:
Lets do that in a separate PR.

Reply:
Ok.